### ISSUE TITLE
Window Size Shrinks on Display Add/Remove

### ISSUE THREAD

**Issue by dsanders11** [a85/POSTMan-Chrome-Extension#10700]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues

### Describe the Issue

Adding or removing a display to the system on Windows will cause the Postman app window to shrink by 1 pixel in both width and height.

### Steps To Reproduce

1. Open Postman app, in a non-maximized state
2. Connect another display

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Native App
- Postman Version: Any
```

### Additional Context?

This is caused by an Electron issue: https://github.com/electron/electron/issues/33049

Root cause is upstream in Chromium.

**Comment by RaisinTen** [comment]
@dsanders11 I think this got fixed a while back. Could you please test out the latest Postman version and confirm?

**Comment by dsanders11** [comment]
@RaisinTen, confirmed as fixed when testing on v9.19.0, closing. Thanks for the reminder.
